custom_md_loader/pages/CMTMigrator.page

```diff
@@ -0,0 +1,256 @@
+								 * Desc     	Migrate Custom Settings/Custom Objects data to Custom Metadata Types records if you have only  <br/> 
+								 *				different Api names in Custom Settings and Custom Metadata Types <br/> 
+								 * @param      	Name of Custom Setting/Custom Objects Api (VAT_Settings_CS__c) <br/> 
+								 * @param      	Name of Custom Metadata Types Api (VAT_Settings__mdt) <br/> 
+								 * @param      	Json Mapping (Sample below) <br/> 
+								 					{         <br/> 
+														"Active__c" : "IsActive__c", <br/> 
+														"Timeout__c" : "GlobalTimeout__c",  <br/>        
+														"EndPointURL__c" : "URL__c", <br/> 
+													} <br/> 
+														 	 <br/> 				
+								 * @return	    <br/> 
+								 *********************************************************/ <br/>  <br/> 
+		                         <br/> 
+								String jsonMapping = '{'+ <br/>         
+									'"Active__c" : "Active__c",'+ <br/> 
+									'"Timeout__c" : "Timeout__c",'+ <br/>         
+									'"EndPointURL__c" : "EndPointURL__c",'+ <br/> 
+									'};'; <br/> 
+						    	MetadataLoader loader = MetadataLoaderFactory.getLoader(MetadataOpType.APEXWRAPPER); <br/>
+						        loader.migrateCustomMapping('VAT_Settings_CS__c', 'VAT_Settings__mdt', jsonMapping); <br/>
+						         <br/>
+		                    </li>
+		                </ul>
+
+		            <br/>
+		                   
+		        </ol>
+		    </apex:pageBlock>
+		                        
+            <apex:pageBlock title="Migrate - AS Is (with object creation)">
+            
+                 <table>
+                     <tr>
+                        <td><b>Custom Setting/Custom Objects Name (From) </b></td>
+                        <td><apex:inputText value="{!csFieldObjCreation}" id="csFieldObjCreation" size="100"/></td>
+                    </tr>  
+                     <tr>
+                        <td><b>Custom Metadata Type Name (To) </b></td>
+                        <td><apex:inputText value="{!cmtFieldObjCreation}" id="cmtFieldObjCreation" size="100"/></td>
+                        
+                    </tr>        
+                     <tr>
+                        <td><b>Operation Type</b></td>
+                        <td> <apex:selectList value="{!opTypeFieldObjCreation}" size="1">
+                                  <apex:selectOptions value="{!objCreationOpTypes}"/>
+                            </apex:selectList>
+                          </td>
+                    </tr>        
+                              
+                </table>
+                                
+                <div draggable="false" >
+                    <br/>
+                        <apex:commandButton value="Migrate - As Is (with object creation)" action="{!migrateAsIsWithObjCreation}" id="migrateAsIsWithObjCreation"/>
+                     <br/>
+                </div>
+            </apex:pageBlock>
+            
+            <apex:pageBlock title="Migrate As Is">
```

Comment:
Could be clearer, such as "Migrate Custom Settings to existing Custom Metadata Type"